plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -133,6 +133,20 @@ public void testCountAggregationView()
                 "VALUES (1)");
     }
 
+    @Test(description = "regression test for https://github.com/trinodb/trino/issues/6696")
+    public void testRepeatCountAggregationView()
+    {
+        BigQuery client = createBigQueryClient();
+
+        String viewName = "test.repeat_count_aggregation_view";
```

Comment:
use randomized name (randomTableSuffix). these tests are likely to be run on shared resources